archetypes/blog-post/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+date: {{ .Date }}
```

Comment:
What format does the date end up looking like? Should we do something like `{{ .Date.Format "2006-01-02" }}` so it ends up being simpler like 2019-07-17?

Reply:
It ends up a `datetime` (with timezone offset), which is actually a good thing IMO, since if you do want to schedule a post to go out after a particular date and time, it's clear how to express that.